webapp/src/widget/menu.scss

```diff
@@ -1,10 +1,10 @@
 .status-dropdown-menu {
-    > .Menu {
+    > .TodoMenu {
```

Comment:
Unrelated to this PR, but it seems wrong to assume that the plugin's `TodoMenu` component will always be a next-level descendant of the element that has `status-dropdown-menu` class name

Reply:
I thought it would be okay because  `position` is applied separately to the TodoMenu classname.
https://github.com/mattermost/mattermost-plugin-todo/blob/a309a7ab63f0ea0ec7b91e7f46b22045e84ee08e/webapp/src/widget/menu.scss#L7-L10

Reply:
Right, but `.status-dropdown-menu` is managed by the core webapp. At any point, the webapp could restructure itsrlf and do one of:
- Remove that class name altogether
- Change its component structure to have a component in between `.status-dropdown-menu` and the plugin's component

Both of these would make it so our plugin styles don't get used. The alternative is to not have the `>` in the selector